clip-geometry/src/main/java/com/esri/arcgismaps/sample/clipgeometry/MainActivity.kt

```diff
@@ -39,6 +39,9 @@ import com.esri.arcgismaps.sample.clipgeometry.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
 
+    private val graphicsOverlay: GraphicsOverlay by lazy { GraphicsOverlay() }
+    private val envelopesOverlay: GraphicsOverlay by lazy { GraphicsOverlay() }
```

Comment:
```suggestion
    // graphics overlay along the Colorado state border
    private val coloradoOverlay: GraphicsOverlay by lazy { GraphicsOverlay() }

    // graphics overlay to contain the clipping envelopes
    private val envelopesOverlay: GraphicsOverlay by lazy { GraphicsOverlay() }
```

Reply:
Yes makes sense to rename to coloradoOverlay for better readability.